mtags/src/main/scala/scala/meta/internal/pc/OverrideCompletions.scala

```diff
@@ -0,0 +1,461 @@
+      val filterText: String,
+      sym: Symbol,
+      val autoImports: List[l.TextEdit],
+      val detail: String
+  ) extends ScopeMember(sym, NoType, true, EmptyTree)
+
+  /**
+   * An `override def` completion to implement methods from the supertype.
+   *
+   * @param name the name of the method being completed including the `_CURSOR_` suffix.
+   * @param t the enclosing template for the class/object/trait we are implementing.
+   * @param pos the position of the completion request, points to `_CURSOR_`.
+   * @param text the text of the original source code without `_CURSOR_`.
+   * @param start the position start of the completion.
+   * @param isCandidate the determination of whether the symbol will be a possible completion item.
+   */
+  case class Override(
+      name: Name,
+      t: Template,
+      pos: Position,
+      text: String,
+      start: Int,
+      isCandidate: Symbol => Boolean
+  ) extends CompletionPosition {
+    val prefix: String = name.toString.stripSuffix(CURSOR)
+    val typed: Tree = typedTreeAt(t.pos)
+    val isDecl: Set[Symbol] = typed.tpe.decls.toSet
+    val range: l.Range = pos.withStart(start).withEnd(pos.point).toLSP
+    val lineStart: RunId = pos.source.lineToOffset(pos.line - 1)
+
+    override def contribute: List[Member] = {
+      if (start < 0) {
+        Nil
+      } else {
+        val overrideMembers = getMembers(
+          typed,
+          range,
+          pos,
+          text,
+          text.startsWith("o", start),
+          true,
+          isCandidate
+        )
+
+        val overrideDefMembers: List[OverrideDefMember] =
+          overrideMembers
+            .filter { candidate =>
+              CompletionFuzzy.matchesSubCharacters(
+                prefix,
+                candidate.filterText
+              )
+            }
+
+        val allAbstractMembers = overrideMembers
+          .filter(_.sym.isAbstract)
+
+        val (allAbstractEdits, allAbstractImports) = toEdits(allAbstractMembers)
+
+        if (allAbstractMembers.length > 1 && overrideDefMembers.length > 1) {
+          val necessaryIndent = if (metalsConfig.snippetAutoIndent()) {
+            ""
+          } else {
+            val amount =
+              allAbstractEdits.head.getRange.getStart.getCharacter
+            " " * amount
+          }
+
+          val implementAll: TextEditMember = new TextEditMember(
+            prefix,
+            new l.TextEdit(
+              range,
+              allAbstractEdits
+                .map(_.getNewText)
+                .mkString(s"\n${necessaryIndent}")
+            ),
+            completionsSymbol("implement"),
+            label = Some("Implement all members"),
+            detail = Some(s" (${allAbstractEdits.length} total)"),
+            additionalTextEdits = allAbstractImports.toList
+          )
+
+          implementAll :: overrideDefMembers
+        } else {
+          overrideDefMembers
+        }
+      }
+    }
+  }
+
+  private def getMembers(
```

Comment:
Extracted the logic to calculate the list of `OverrideDefMember` so that this method can be used from both
- `case class Override`
- `implementAll()`